kalite/inline/static/js/inline/views.js

```diff
@@ -24,7 +24,8 @@ var ButtonView = BaseView.extend({
                 var narr = self.model.attributes;
 
                 //translate narrative into build options for introjs
-                var options = self.parseNarrative(narr);
+                var before_showing = [];
+                var options = self.parseNarrative(narr, before_showing);
```

Comment:
Instead of passing in an object as a parameter and modifying it, I think it's preferable to include it in the returned object. Something like
```js
var options = self.parseNarrative(narr);
var before_showing = options.before_showing;
```